package/src/components/Attachment/Card.tsx

```diff
@@ -180,11 +201,18 @@ const CardWithContext = <
       {CardCover && <CardCover {...props} />}
       {uri && !CardCover && (
         <View>
-          <Image
+          <ImageBackground
+            imageStyle={styles.cardCover}
             resizeMode='cover'
             source={{ uri: makeImageCompatibleUrl(uri) }}
             style={[styles.cardCover, cover, stylesProp.cardCover]}
-          />
+          >
+            {isVideo ? (
```

Comment:
`isVideo && !lastAttachment.og_scrape_url` will solve your problem. I think this came from the video PR. Sorry for that!

Reply:
Fixed this - #1434

Reply:
I am merging the develop into this

Reply:
You need to make similar change here @Enigma-I-am . The PR that @khushal87 shared only updates `Reply` component

Reply:
This change made the play button disappear totally. went with `isVideo && og_scrape_url` instead

Reply:
Ohhh... I understand now you meant `isVideo && lastAttachment.og_scrape_url` and not `isVideo && !lastAttachment.og_scrape_url`